packages/browser/src/index.test.ts

```diff
@@ -4,26 +4,49 @@ import LogtoClient, { LogtoClientError, LogtoSignInSessionItem } from '.';
 
 const clientId = 'client_id_value';
 const endpoint = 'https://logto.dev';
+
 const authorizationEndpoint = `${endpoint}/oidc/auth`;
+const tokenEndpoint = `${endpoint}/oidc/token`;
 const endSessionEndpoint = `${endpoint}/oidc/session/end`;
 const revocationEndpoint = `${endpoint}/oidc/token/revocation`;
+const jwksUri = `${endpoint}/oidc/jwks`;
+const issuer = 'http://localhost:443/oidc';
+
 const mockedCodeVerifier = 'code_verifier_value';
 const mockedState = 'state_value';
 const mockedSignInUri = `${authorizationEndpoint}?foo=bar`;
```

Comment:
what is this foo=bar for?

Reply:
Nothing. (just add some parameters.)

Updated to a URI like a real one. PTAL